config/testgrids/conformance/conformance-all.yaml

```diff
@@ -5,6 +5,7 @@ dashboard_groups:
     - conformance-apisnoop
     - conformance-ec2
     - conformance-gce
+    - conformance-azure
```

Comment:
it probably makes more sense to have a conformance-kops dashboard, that will have a lot more to do with if conformance works than the cloud provider used. ec2 and GCE here are generally bad shorthands for specific tools